README.md

```diff
@@ -227,22 +226,39 @@ for advice or just jump in and submit an
 [pull request](https://github.com/PyFstat/PyFstat/compare).
 
 Here's what you need to know:
+* As a developer, you should ideally install directly from a git clone,
+  with either `pip install -e .[dev]` into a venv
+  or creating a conda environment from the
+  [pyfstat-dev.yml file](https://github.com/PyFstat/PyFstat/blob/dev-cleanup/etc/pyfstat-dev.yml)
```

Comment:
Please also run, just once after installing:
  ```
  pre-commit install
  ```

  This sets up everything for automated code quality tests (see below)
  to be checked for you at every commit.